libs/polysemy-wire-zoo/src/Wire/Sem/Concurrency/Sequential.hs

```diff
@@ -4,6 +4,9 @@ import Imports
 import Polysemy
 import Wire.Sem.Concurrency
 
+------------------------------------------------------------------------------
+
+-- | Safely perform "concurrency" by doing it sequentially.
 sequentiallyPerformConcurrency :: Sem (Concurrency safe ': r) a -> Sem r a
```

Comment:
Should this have been `'Safe` instead of `safe` in the type `:: Sem (Concurrency safe ': r) a -> Sem r a`?

Reply:
@isovector , I guess this has slipped your attention because I merged this before you got a chance to reply. Here's just a reminder.